control/controlclient/direct.go

```diff
@@ -299,6 +302,8 @@ func (c *Direct) doLogin(ctx context.Context, opt loginOpt) (mustRegen bool, new
 	hi := c.hostinfo.Clone()
 	backendLogID := hi.BackendLogID
 	expired := c.expiry != nil && !c.expiry.IsZero() && c.expiry.Before(c.timeNow())
+	serverURL := c.serverURL
+	httpc := c.httpc
```

Comment:
`c.httpc` isn't guarded by `c.mu` so this is misleading. Just go back to passing `c.httpc` on line 334 like before.

Reply:
Done